protocol/src/main/java/com/amazon/opendistroforelasticsearch/sql/protocol/response/format/FlatResponseFormatter.java

```diff
@@ -100,12 +109,12 @@ private String getHeaderLine(QueryResult response, boolean sanitize) {
     }
 
     /**
-     * Sanitize CSV headers because Elasticsearch allows special character present in field names.
+     * Sanitize headers because Elasticsearch allows special character present in field names.
      */
     private List<String> sanitizeHeaders(List<String> headers) {
       return headers.stream()
           .map(this::sanitizeCell)
-          .map(cell -> quoteIfRequired(INLINE_SEPARATOR, cell))
+          .map(cell -> quoteIfRequired(separator, cell))
```

Comment:
For now both sanitize and escape logic is controlled by `sanitize` field, is this why we want `escape=true` by default even for RAW formatter? Should we always do escape despite of `escape`?